build/regressions.config.yml

```diff
@@ -39,6 +39,28 @@ sources:
         - sebastienros
         - agocke
       template: build
+  - name: MVC 
+    table: MvcBenchmarks
+    rules:
+      - include: ApiCrud
+        labels: 
+          - area-mvc
+      - exclude: ulib
```

Comment:
These two rules can be removed, it's an example of the explanation I gave about these rules, to exclude specific scenarios (of group of) from the bot.

Reply:
Removed